doc/develop/modules.rst

```diff
@@ -569,6 +569,36 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+SBOM integration
```

Comment:
Should we turn this around.

Instead of having `SBOM integration`, then describe what content is (example: `Vulnerability monitoring`), and then instead describe that this information will be included in the sbom.

The vulnerability monitoring / information can be of interest, even without an SBOM.

(Just a thought, not a request for a change)